web-app/package.json

```diff
@@ -24,32 +24,33 @@
   "dependencies": {
     "compression": "^1.7.1",
     "dotenv": "^8.2.0",
-    "google-protobuf": "^3.6.1",
+    "google-protobuf": "^3.11.2",
     "grpc-web": "^1.0.0",
     "js-cookie": "^2.2.1",
-    "polka": "next"
+    "polka": "^1.0.0-next.9",
+    "purgecss": "^1.4.2",
+    "purgecss-webpack-plugin": "^1.6.0"
```

Comment:
I'm 99% sure `purgecss` and `purgecss-webpack-plugin` should have been `devDependencies`, I'll fix that and regenerate the lockfile later today.